EnhancedMediaPlayer/MediaPlayerViewController.swift

```diff
@@ -40,6 +42,20 @@ public class MediaPlayerViewController: UIHostingController<MediaPlayerView> {
                 case .forward: self.forward()
             }
         }
+
+        viewModel.onSeekDoubleTap = { [weak self] in
+            guard let self else { return }
+
+            Task {
+                self.tapTimer?.invalidate()
+
+                self.viewModel.isTappingSeek = true
+
+                self.tapTimer = Timer.scheduledTimer(withTimeInterval: Constants.seekTapInterval, repeats: false) { _ in
+                    self.viewModel.isTappingSeek = false
+                }
```

Comment:
See, you're already setting a timer here, I don't see the need to handle those things in the View

Reply:
This timer handles the time between a double tap and a sequential single tap (and also taps after this single tap). Just like YouTube`s behavior. The other timer handles animation of forward and backward icons: the icon shows up quickly and ease fade out